htmd/parameterization/cli.py

```diff
@@ -308,6 +308,45 @@ def _get_initial_parameters(mol, args):
     return mol, parameters
 
 
+def _fit_initial_charges(mol, args):
+
+    from htmd.charge import fitGasteigerCharges
+    from htmd.parameterization.util import guessBondType
+
+    logger.info('=== Initial atomic charge fitting ===')
+
+    if args.charge_type == 'None':
+        logger.info('Initial atomic charges are taken from {}'.format(args.filename))
+
+    elif args.charge_type in ('Gasteiger', 'AM1-BCC', 'ESP'):
+        if args.min_type == 'mm':
+            logger.info('Method: Gasteiger')
+
+            # TODO move to _prepare_molecule
+            if np.any(mol.bondtype == "un"):
+                logger.info('Guessing bond types')
+                mol = guessBondType(mol)
+
+            mol = fitGasteigerCharges(mol)
+
+            # Print the initial charges
+            logger.info('Initial atomic charges:')
+            for name, charge in zip(mol.name, mol.charge):
+                logger.info('   {:4s}: {:6.3f}'.format(name, charge))
+            logger.info('Molecular charge: {:6.3f}'.format(np.sum(mol.charge)))
+
+        elif args.min_type in ('None', 'qm'):
+            logger.info('Initial atomic charges are not required')
+
+        else:
+            raise ValueError()
```

Comment:
Ok anyway I'll accept the PR like this but what is with these two empty valuerrors? I assume they won't be reached due to the argparser but then why add them? If you keep them include some message

Reply:
Yes, these are not suppose to reached, just safeguards, if we change something in the future.

Reply:
There is no such thing as `CodeConsistencyError`, I used `ValueError`

Reply:
It's called `AssertionError` or `assert` ;)

Reply:
OK! I will change...